variables.tf

```diff
@@ -221,247 +221,30 @@ variable "aws_auth_additional_labels" {
   type        = map(string)
 }
 
-# KUBERNETES ADDONS VARIABLES
-variable "enable_emr_on_eks" {
-  type        = bool
-  default     = false
-  description = "Enabling EMR on EKS Config"
-}
-
-variable "emr_on_eks_teams" {
-  description = "EMR on EKS Teams configuration"
-  type        = any
-  default     = {}
-}
-
-#-----------CLUSTER AUTOSCALER-------------
-variable "cluster_autoscaler_enable" {
-  type        = bool
-  default     = false
-  description = "Enabling Cluster autoscaler on eks cluster"
-}
-
-variable "cluster_autoscaler_helm_chart" {
-  type        = any
-  default     = {}
-  description = "Cluster Autoscaler Helm Chart Config"
-}
-
-#-----------PROMETHEUS-------------
+#-----------Amazon Managed Prometheus-------------
 variable "aws_managed_prometheus_enable" {
   type        = bool
   default     = false
   description = "Enable AWS Managed Prometheus service"
 }
 
-variable "aws_managed_prometheus_workspace_name" {
+variable "aws_managed_prometheus_workspace_id" {
   type        = string
-  default     = "aws-managed-prometheus-workspace"
+  default     = ""
```

Comment:
should we provide a default?

Reply:
Yes, it's by default empty string. If users doesn't provide any input then we are building a name using the cluster id in `locals.tf`. 

 ` aws_managed_prometheus_workspace_id           = format("%s-%s", "amp-workspace-", module.aws_eks.cluster_id)`